### ISSUE TITLE
Expose raw theme colors?

### ISSUE THREAD

**Issue by jlewin** [Railly/tinte#35]
I'd really appreciate being able to hit a theme url, and pull down the color values for the theme. Any reason you wouldn't want to expose this data for use? Currently I can hit the homepage, browse to a theme I like and see its effect, but my only options are to export to vsix and vscode. If I could simply add ".json" to the theme url, and get at the underlying theme values, I could use these in creative ways, similar to if I take the time to extract them from vscode or the vxis.

https://tinte.railly.dev/t/rec_cskhdctqrj660d2fe5lg.json
  or 
https://tinte.railly.dev/t/json/rec_cskhdctqrj660d2fe5lg

Thought or concerns?

**Comment by Railly** [comment]
Yeah, it should be either copyable or exposed in some way. I just created a ticket to look into this. Thanks!

**Comment by jlewin** [comment]
Great to hear. I ended up writing a parser for vsix files, which fits my immediate needs, but I'd still prefer to use the simplified color values from the themes. 

Any chance this would be a good community contribution task? I could tackle it if you have guidance on how you'd like it implemented. Otherwise, I'll just keep an eye out for a future update.
